src/backend/server.js

```diff
@@ -131,6 +132,55 @@ app.post('/log', wrap(async (req, res) => {
   res.send('')
 }))
 
+app.post('/password_reset', wrap(async (req, res) => {
+  // All errors in this endpoint will result in a 200 response.
+  // This prevents people from mining emails through brute
+  // force.
+  if(!req.body.email) {
+    return res.status(200).end();
+  }
+
+  let user = await models.User.findOne({ where: {email:req.body.email} });
+
+  if(!user) {
+    return res.status(200).end();
+  }
+
+  console.log(req.body.email)
```

Comment:
Extraneous debug.  I actually added a log library (log.debug) that we could use and then debug logs won't make their way into production logs.